lua/lint/linters/prisma_lint.lua

```diff
@@ -0,0 +1,38 @@
+local binary_name = "prisma-lint"
```

Comment:
Would it be better to call this file `prisma-lint` (with a hyphen instead of an underscore) instead to match? Not sure what's idiomatic here.

Reply:
So far the files tend to use underscore if the binary has a hyphen.

Could you also add the linter to the readme?